src/SourceConstraints.hs

```diff
@@ -48,18 +53,24 @@ runSourceConstraints :: [CommandLineOption]
                      -> ModSummary
                      -> HsParsedModule
                      -> Hsc HsParsedModule
-runSourceConstraints _options ModSummary{ms_location = ModLocation{..}} parsedModule = do
+runSourceConstraints options ModSummary{ms_location = ModLocation{..}} parsedModule = do
   dynFlags <- getDynFlags
 
+  localModules <- mapM parseLocalModule (pack <$> options)
+
   when (allowLocation ml_hs_file) $
     liftIO
       . printOrThrowWarnings dynFlags
       $ warnings Context{..} (hpm_module parsedModule)
 
   pure parsedModule
+
```

Comment:
@mbj To be consistent with other `where` clauses, this blank line should be removed.

Reply:
Bump